app/routes/app.tsx

```diff
@@ -32,12 +32,12 @@ export const action = async ({ request }: ActionFunctionArgs) => {
 
 export default function AppLayout() {
 	return (
-		<div className="grid grid-cols-5 h-screen">
-			<div className="col-span-1 h-screen border-e bg-zinc-50 dark:(bg-neutral-900 border-neutral-700) flex flex-col">
+		<div className="flex h-screen">
+			<div className="w-78 h-screen border-e bg-zinc-50 dark:(bg-neutral-900 border-neutral-700) flex flex-col">
```

Comment:
If we're setting a width on this, then we don't need to send the tabs below the logo anymore.

Also add a `min-w-78` so that it can't shrink below that